it/uk/gov/hmrc/submissiontracker/controllers/SandboxSubmissionTrackerControllerISpec.scala

```diff
@@ -13,39 +13,39 @@ class SandboxSubmissionTrackerControllerISpec extends BaseISpec with FileResourc
   "GET /sandbox/tracking/:id/:idType" should {
 
     "return valid tracking data from sandbox with accept header" in {
-      val response = await(wsUrl(s"/tracking/$nino/$idType")
+      val response = await(wsUrl(s"/tracking/$nino/$idType/$journeyIdUrlVar")
```

Comment:
should be a ? not a / before the variable. Or just no / if the ? is being added in the varaible